### ISSUE TITLE
Max Prompt Token Usage

### ISSUE THREAD

**Issue by ericdrgn** [matrixgpt/matrix-chatgpt-bot#140]
I am seeing all requests through the bot have max token usage on prompts. Even making a simple prompt of "hi" triggers a prompt token usage of 3000 tokens. Running the same thing through a simple terminal curl request results in a total of 8 tokens used. What's going on? This wasn't happening yesterday.

**Comment by ericdrgn** [comment]
Confirmed usage in playground is also working fine, it seems the only time token usage is off the charts for a prompt of "hi" is when using this matrix bot. I have attached a screenshot of the usage displayed in the OpenAI dashboard. I ran a prompt of "hi" through the OpenAI Playground, then the Matrix bot, then through a standard curl command. Is no one else seeing this issue? I haven't changed anything and looking through the history of requests it started this morning.

![image](https://user-images.githubusercontent.com/60805441/225731236-1c328562-9689-4ab2-a537-fcbc68cd2c9b.png)
